public/views/dashboard.html

```diff
@@ -0,0 +1,126 @@
+<div ng-controller="DashboardController">
+    <body>
+      <!-- Navigation -->
+      <div class="container-fluid">
+          <div class="row">
```

Comment:
2 spaces instead of 4
Ensure the spacing in this file is 2 spaces instead of 4. Some lines have 4 spaces